react/index.tsx

```diff
@@ -64,38 +50,12 @@ const OrderPlaced: FC = () => {
     // 'any' needed because graphql error type doesn't have 'extensions' prop
     (error as any)?.extensions?.response?.status === 403
   ) {
-    // if query errored display an error alert
-    return (
-      <ErrorMessage
-        icon={<Forbidden />}
-        errorId={messages.notLoggedTitle.id}
-        messageId={messages.notLoggedMessage.id}
-      >
-        <a href={`/login?returnUrl=${window.location.href}`}>
-          <Button>
-            <FormattedMessage id="store/go-to-login" />
-          </Button>
-        </a>
-      </ErrorMessage>
-    )
+    return <ForbiddenError />
   }
 
   // not found error
-  /** if query resulted in an invalid orderGroup display an error alert*/
   if (data?.orderGroup?.orders == null) {
-    return (
-      <ErrorMessage
-        icon={<NotFound />}
-        errorId={messages.invalidTitle.id}
-        messageId={messages.invalidMessage.id}
-      >
-        <a href="/">
-          <Button>
-            <FormattedMessage id="store/go-to-home" />
-          </Button>
-        </a>
-      </ErrorMessage>
-    )
+    return <InvalidError />
   }
 
   const { orderGroup }: { orderGroup: OrderGroup } = orderGroupQuery as any // data
```

Comment:
Useful comment?

Reply:
The comment is the actual production value, just commented it out for testing/mocking purposes. Will uncomment in the future